src/main/java/org/olf/folio/order/MarcToJson.java

```diff
@@ -0,0 +1,408 @@
+package org.olf.folio.order;
```

Comment:
Are you planning on eventually using this class (calling it from `OrderImport`) or is it just an artifact of the refactoring that should be removed?

Reply:
I created it so that I could test the JSON that is being created without posting to FOLIO.  It's not part of the app, but I don't want to remove it as I find it useful for development.